internal/credential/model.go

```diff
@@ -19,6 +19,7 @@ type Container struct {
 	ID            string
 	Credential    *credential.VerifiableCredential
 	CredentialJWT *keyaccess.JWT
+	Revoked       bool
```

Comment:
I'm wondering if we should only expose `revoked` or allow this to be a `map[string]bool` of all possible statuses a credential can have. possible a later improvement.

Reply:
for now I'll add revoked and suspended and in the future if there is a need we can make it generic

Reply:
created issue -https://github.com/TBD54566975/ssi-service/issues/177